### PR TITLE
Add cdi 3.0 beans.xml namespace test

### PR DESCRIPTION
**Pull request by tevans78**
Tests that a simple CDI app will start using the new Jakarta EE 9 namespaces in a beans.xml

### CODE REVIEW

**Comment by tevans78** [comment]
#build

File: dev/fattest.simplicity/src/com/ibm/websphere/simplicity/BeansAsset.java [hunk 1]
```diff
@@ -42,32 +42,72 @@
  *              .addAsWebInfResource(beans, "beans.xml");
  * </code>
  * </pre>
+ *
```
Comment (hutchig) [inline dev/fattest.simplicity/src/com/ibm/websphere/simplicity/BeansAsset.java:45]: Please raise an issue rather than check in TODOs in code.

**Review by hutchig — CHANGES_REQUESTED** [review]

File: dev/fattest.simplicity/src/com/ibm/websphere/simplicity/BeansAsset.java (hunk 1 above)
Comment (tevans78) [reply dev/fattest.simplicity/src/com/ibm/websphere/simplicity/BeansAsset.java:45]: Good point - https://github.com/OpenLiberty/open-liberty/issues/16057

**Review by benjamin-confino — APPROVED** [review]
All looks good. I'd consider renaming SimpleBean to UnannotatedBean but that's not worth withholding approval for.

**Comment by hutchig** [comment]
Feel free to merge if you have a plan to address my comment as I am offline this PM on PTO.
"TODOs" show up in the eclipse 'things to do' marker list for everyone and people use them for there 
own sets of things to fix before pushing also
one never knows if it was an omission or just a 'nice to have', either way, we don't leave them in the code
in OpenLiberty.

**Comment by tevans78** [comment]
@hutchig I'm running another PB anyway so I removed the TODOs for you
#build

**Review by hutchig — APPROVED** [review]

**Comment by tevans78** [comment]
Despite `wlp.lib.extract_fat` having failed in both personal builds, I don't think there is any way this change is related. There is also an open defect for it: 276597
#run-libby-bot

**Pull request merged**